The example for C-BITFLAGS is broken

The example for the `C-BITFLAGS` section doesn't compile (the constants are scoped): 

```
   Compiling playground v0.0.1 (file:///playground)
error[E0425]: cannot find value `FLAG_A` in this scope
  --> src/main.rs:13:26
   |
13 |     if settings.contains(FLAG_A) {
   |                          ^^^^^^ not found in this scope

error[E0425]: cannot find value `FLAG_B` in this scope
  --> src/main.rs:16:26
   |
16 |     if settings.contains(FLAG_B) {
   |                          ^^^^^^ not found in this scope

error[E0425]: cannot find value `FLAG_C` in this scope
  --> src/main.rs:19:26
   |
19 |     if settings.contains(FLAG_C) {
   |                          ^^^^^^ not found in this scope

error[E0425]: cannot find value `FLAG_A` in this scope
  --> src/main.rs:25:7
   |
25 |     f(FLAG_A | FLAG_C);
   |       ^^^^^^ not found in this scope

error[E0425]: cannot find value `FLAG_C` in this scope
  --> src/main.rs:25:16
   |
25 |     f(FLAG_A | FLAG_C);
   |                ^^^^^^ not found in this scope

error: aborting due to 5 previous errors

error: Could not compile `playground`.

To learn more, run the command again with --verbose.
```

--- a/src/type-safety.md
+++ b/src/type-safety.md
@@ -106,19 +106,19 @@ bitflags! {
 }
 
 fn f(settings: Flags) {
-    if settings.contains(FLAG_A) {
+    if settings.contains(Flags::FLAG_A) {
         println!("doing thing A");
     }
-    if settings.contains(FLAG_B) {
+    if settings.contains(Flags::FLAG_B) {
         println!("doing thing B");
     }
-    if settings.contains(FLAG_C) {
+    if settings.contains(Flags::FLAG_C) {
         println!("doing thing C");
     }
 }
 
 fn main() {
-    f(FLAG_A | FLAG_C);
+    f(Flags::FLAG_A | Flags::FLAG_C);
 }
 ```
 

